src/main/java/core/basesyntax/Constant.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Constant {
+    private final int ballNumber = new Random().nextInt(100);
+
+    public int getBallNumber() {
+        return ballNumber;
+    }
+
+}
```

Comment:
useless class
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
```

Reply:
not fixed
remove this class